pkg/controller/servicemesh/memberroll/controller_test.go

```diff
@@ -464,6 +464,33 @@ func TestReconcileNamespacesIgnoresControlPlaneNamespace(t *testing.T) {
 	assert.DeepEquals(configuredMembers, []string{appNamespace}, "reconcileNamespaces returned an unexpected configuredMembers list", t)
 }
 
+func TestReconcileWorksWithMultipleNamespaces(t *testing.T) {
+	roll := newDefaultMemberRoll()
+	addOwnerReference(roll)
+	roll.Spec.Members = []string{appNamespace, appNamespace2}
+	roll.ObjectMeta.Generation = 2
+	roll.Status.ObservedGeneration = 1
+	controlPlane := markControlPlaneReconciled(newControlPlane(""), meshVersionDefault, operatorVersionDefault)
```

Comment:
Nit: moving this line to the top would make this setup code a bit cleaner (create the controlePlane, then the roll).

Reply:
Done